spec/responses/stash_api/datasets_controller_spec.rb

```diff
@@ -54,6 +54,124 @@ module StashApi
       end
 
     end
+
+    describe '#index' do
+
+      before(:each) do
+        # these tests are very similar to tests in the model controller for identifier for querying this scope
+        @user1 = create(:user, tenant_id: 'ucop', role: nil)
+        @user2 = create(:user, tenant_id: 'ucop', role: 'admin')
+        @user3 = create(:user, tenant_id: 'ucb', role: 'superuser')
+
+        @identifiers = []
+        0.upto(7).each { |_i| @identifiers.push(create(:identifier)) }
+
+        @resources = [create(:resource, user_id: @user1.id, tenant_id: @user1.tenant_id, identifier_id: @identifiers[0].id),
```

Comment:
You should be able to use the resources factory traits for this. For example `create(:resource_published)` or `create(:resource_submitted)`. 

You might need to update them to use your new `:curation_activity_no_callbacks` factory though. I would think that's useful for 90% of the tests anyway

Maybe something to come to though after MVP

Reply:
Good info, thanks, Brian.